types/types.go

```diff
@@ -444,15 +444,20 @@ type grafanaOutputConfig struct {
 type YandexOutputConfig struct {
 	AccessKeyID     string
 	SecretAccessKey string
+	Endpoint        string
```

Comment:
Are you sure we can move this setting here? It creates a breaking change for users.

Reply:
Yes, you are right. I misunderstood. Fixed